app/facades/foods_facade.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsFacade
+  def self.service
+    @_service ||= FoodsService.new
```

Comment:
Typically, service methods are class methods so that you don't have to instantiate them like this.